src/components/campaigns/CampaignInfo.tsx

```diff
@@ -136,7 +149,8 @@ export default function CampaignInfo({ campaign }: Props) {
           <strong>{t('campaigns:campaign.profile')}</strong>Спешна
         </Typography> */}
         <Typography variant="subtitle2" component="p" className={classes.campaignText}>
-          <strong>{t('campaigns:campaign.status')}</strong> {campaign.state}
+          <strong>{t('campaigns:campaign.status')}</strong>{' '}
```

Comment:
you can translate the status dynamically in this way:
{t(`campaigns:campaign-status.${campaign.state}`)}

and you don't need the  "const status " from above